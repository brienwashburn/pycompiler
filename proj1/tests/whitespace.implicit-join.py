month_names = ['J', 'F', 'M',
               'A', 'M', 'J',
               'J', 'A', 'S',
               'O', 'N', 'D']
