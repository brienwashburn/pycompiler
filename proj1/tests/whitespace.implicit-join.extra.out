(KEYWORD if)
(KEYWORD True)
(PUNCT ":")
(NEWLINE)
(INDENT)
(ID "print")
(PUNCT "(")
(LIT 3)
(PUNCT ")")
(NEWLINE)
(DEDENT)
(ENDMARKER)
